mixin-metrics.go

```diff
@@ -0,0 +1,319 @@
+
+// parses Prom metrics from dashboard JSON and YAML rules
```

Comment:
```suggestion
// Command mixin-metrics parses Prom metrics from dashboard JSON and YAML rules
```

Style nit: program commands should start with `// Command <program name>` and non-main package comments should start with `// Package <package name>`